turbine_models/parser.py

```diff
@@ -155,15 +166,25 @@ def specs(self, index, group="onshore"):
             spec_data = None
         return spec_data
 
+    
     def find_group_for_turbine(self,turbine_name):
         
+        # check if turbine name was provided as full name
         for group in self.groups:
             turbines_in_group = self.turbines(group)
             if any(k.lower()==turbine_name.lower() for k in list(turbines_in_group.values())):
```

Comment:
This is a bigger question, so doesn't need to be resolved here at all: should we normalize all names to just be lowercase? It seems like there is a decent amount of string normalization, but it could be easier to just support it off the bat.